tests/benchmarks/model_benchmarks/test_pytorch_base.py

```diff
@@ -250,16 +250,36 @@ def test_pytorch_empty_cache():
     # Register mnist benchmark.
     BenchmarkRegistry.register_benchmark('pytorch-mnist', PytorchMNIST)
 
+    # Get initial memory reserved
+    init_res_memory = torch.cuda.memory_reserved()
+    assert (init_res_memory >= 0)
```

Comment:
Can memory_reserved() return a negative value? or this assert is unnecessary.

Reply:
left the check as the return was an int type.
But based on the [impl](https://github.com/pytorch/pytorch/blob/main/torch/cuda/memory.py#L204-L209), it is guaranteed to be a non negative integer.
removing the redundant check.